python/src/aiconfig/schema.py

```diff
@@ -691,7 +691,9 @@ def add_output(self, prompt_name: str, output: Output, overwrite: bool = False):
         prompt = self.get_prompt(prompt_name)
         if not prompt:
             raise IndexError(f"Cannot out output. Prompt '{prompt_name}' not found in config.")
-        if overwrite or not output:
+        if not output:
+            raise Exception(f"Cannot add output to prompt '{prompt_name}'. Output is empty.")
```

Comment:
nit: `output is not defined`

Reply:
Also let's make this a `ValueError` instead of generalized `Exception`

Reply:
Done!